README.md

```diff
@@ -1,121 +1,134 @@
+### Webpack installation
 
-#### Serverside usage with Webpack
+The Ably Pub/Sub SDK includes support for Webpack compiling browsers.
 
-If you are using a version older than 1.2.5 you will need to add 'ably' to `externals` in your Webpack config to exclude it from Webpack processing, and require and use it in as a external module using require('ably') as above.
+<details>
+<summary>Webpack installation details.</summary>
 
-### For browsers
+If you're compiling for the browser, Webpack will resolve `ably` from `node_modules` automatically when included in your `package.json`. You can then:
 
-Include the Ably library in your HTML:
-
-```html
-<script src="https://cdn.ably.com/lib/ably.min-2.js"></script>
+```javascript
+require('ably');
+// or, for ES6/TypeScript:
+import * as Ably from 'ably';
 ```
 
-The Ably client library follows [Semantic Versioning](http://semver.org/). To lock into a major or minor version of the client library, you can specify a specific version number such as https://cdn.ably.com/lib/ably.min-2.js for all v2._ versions, or https://cdn.ably.com/lib/ably.min-2.6.js for all v2.6._ versions, or you can lock into a single release with https://cdn.ably.com/lib/ably.min-2.6.3.js. Note you can load the non-minified version by omitting `.min` from the URL such as https://cdn.ably.com/lib/ably-2.js. See https://github.com/ably/ably-js/tags for a list of tagged releases.
+With `target: 'browser'`, Webpack uses the browser-compatible CommonJS build by default.
+
+If needed (for example with custom targets):
+
+- Webpack 5: To resolve Ably correctly, add this alias to your webpack configuration:
 
-For usage, jump to [Using the Realtime API](#using-the-realtime-api) or [Using the REST API](#using-the-rest-api).
+  ```javascript
+  alias: {
+    ably: path.resolve(__dirname, 'node_modules/ably/build/ably.js'),
+  }
+  ```
+- Webpack < 5: For older versions, import the pre-built bundle directly:
 
-#### Using Webpack
+  ```javascript
+  import * as Ably from 'ably/build/ably.js';
+  ```
+</details>
 
-(This applies to using Webpack to compile for a browser; for Node.js, see [Serverside usage with Webpack](#serverside-usage-with-webpack))
+## Usage
 
-Webpack will search your `node_modules` folder by default, so if you include `ably` in your `package.json` file, when running Webpack the following will allow you to `require('ably')` (or if using typescript or ES6 modules, `import * as Ably from 'ably';`). If your Webpack target is set to 'browser', this will automatically use the browser commonjs distribution.
+The following code connects to Ably's realtime messaging service, subscribes to a channel to receive messages, and publishes a test message to that same channel.
 
-If that doesn't work for some reason (e.g. you are using a custom Webpack target), you can use one of the solutions below depending on your Webpack version:
+```javascript
+const realtimeClient = new Ably.Realtime({ key: '', clientId: 'a-client' });
```

Comment:
I think the string examples we have here look a little odd, would suggest the following:
```suggestion
const realtimeClient = new Ably.Realtime({ key: 'your-ably-api-key', clientId: 'me' });
```

Reply:
Also I think this would look a bit nicer with comments, eg:

```typescript
// Create a new realtime client
const realtimeClient = new Ably.Realtime({ key: 'your-ably-api-key', clientId: 'me' });

// Wait for the client to become connected
await realtimeClient.connection.once('connected');
console.log('Connected to Ably');

// Get a reference to the 'test-channel' channel
const channel = realtimeClient.channels.get('test-channel');

// Subscribe a listener to fire on all events on the channel
await channel.subscribe((message) => {
  console.log(`Received message: ${message.data}`);
});

// Publish a test message to the channel
await channel.publish('test-event', 'Hello, world!');
```